doc/reference/modules/query_linq.xml

```diff
@@ -65,7 +67,9 @@ using NHibernate.Linq;]]></programlisting>
     <para>&nbsp;</para>
 
     <para>
-      A client timeout for the query can be defined.
+      A client timeout for the query can be defined. As most others NHibernate specific features for
+      Linq, this is available through an extension defined in <literal>NHibernate.Linq</literal>
+      namespace.
```

Comment:
Most other features were at least already mentioning they were extensions methods without precising again the namespace, but I think it should be enough this way.